tests/struct.lisp

```diff
@@ -713,3 +713,29 @@
         (setf secs 100 usecs 200)
         (values (mem-ref psecs :long) (mem-ref pusecs :long))))
   100 200)
+
+;;; regression test: if slots use :offset, struct should still be big
+;;; enough to hold all of the slots.
+(defcstruct struct.offsets
+  (a :int32 :offset 0 :count 4)
+  (b :int32)
+  (c :uint8 :offset 0)
+  (d :uint8)
+  (e :uint8)
+  (f :uint8 :offset 6)
+  (g :uint32 :offset 0))
+
+(deftest struct.offset.1
+    (foreign-type-size '(:struct struct.offsets))
+  20)
+
+;; Add some tests for :offset in general
+(deftest struct.offset.2
+    (values (cffi:foreign-slot-offset '(:struct struct.offsets) 'a)
```

Comment:
No need for the package prefix, I guess. We don't use it elsewhere.

Reply:
oops, added it to get autodoc since i hadn't loaded the package, and forgot to remove it